diff --git i/CMakeLists.txt w/CMakeLists.txt
index 44ce06f..70f1568 100644
--- i/CMakeLists.txt
+++ w/CMakeLists.txt
@@ -21,7 +21,7 @@ list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/cmake")
 set(GAFRO_PREFIX_PACKAGE share/${PROJECT_NAME}/cmake)
 set(GAFRO_PREFIX_INCLUDE include/gafro)
 
-find_package(Eigen3 3.4 REQUIRED)
+# find_package(Eigen3 3.4 REQUIRED)
 
 add_subdirectory(src/${PROJECT_NAME})
 add_subdirectory(src/${PROJECT_NAME}_control)
